hack/common

```diff
@@ -188,6 +188,34 @@ function push_image() {
     skopeo copy --dest-tls-verify=false docker-daemon:"$1" docker://"$2"
 }
 
+# debug_print [max_count [interval]]
+function debug_print () {
+  local filename=$(basename "$0")
+  local max=${1:-120}
+  local interval=${2:-10}
+  while [ $max -gt 0 ];
+  do
+    clopod=$( oc -n ${NAMESPACE} get pods -o name | egrep "cluster-logging-operator" ) || :
+	clo=$( echo $clopod | awk -F'/' '{print $2}' )
```

Comment:
```bash
clopod=$( oc -n ${NAMESPACE} get pods -l name=cluster-logging-operator -o jsonpath='{.items[0].metadata.name}' )
```
not sure if using `-l name=cluster-logging-operator` will return a list or a single item, since there is only 1 - if it is the latter, then use `-o jsonpath='{.metadata.name}'`

Reply:
It works just fine.  Thanks!
```
clo=$( oc -n ${NAMESPACE} get pods -l name=cluster-logging-operator -o jsonpath='{.items[0].metadata.name}' )
echo $clo
cluster-logging-operator-54644647d-rr4sj

```